LearningMissionSimulation/ObjectGenerator.cs

```diff
@@ -132,7 +132,9 @@ public static List<Classroom> GenerateClassroomList()
         public static Module GenerateModule(Guid subjectId)
         {
             //not implemented yet!
-            return null;
+            return new Module(subjectId: subjectId, moduleLevel: AttributeGenerator.GetModuleLevel(), prerequisiteList: GenerateModuleList(),
+                              name: AttributeGenerator.GetFirstName(), description: AttributeGenerator.GetUnitDescprition(UnitType.Department),
```

Comment:
1. You should use GetUnitName(UnitType.Module) instead of GetFirstName()
2. You should use GetUnitDescprition(UnitType.Module) instead of GetUnitDescprition(UnitType.Department)